doc/classes/TileSet.xml

```diff
@@ -591,7 +591,7 @@
 			Tile coordinates layout where both axis stay consistent with their respective local horizontal and vertical axis.
 		</constant>
 		<constant name="TILE_LAYOUT_STACKED_OFFSET" value="1" enum="TileLayout">
-			Same as [code]TILE_LAYOUT_STAKED[/code], but the first half-offset is negative instead of positive.
+			Same as [code]TILE_LAYOUT_STACKED[/code], but the first half-offset is negative instead of positive.
```

Comment:
```suggestion
			Same as [constant TILE_LAYOUT_STACKED], but the first half-offset is negative instead of positive.
```

Reply:
Hm, quick grep and referencing constants like this can be applied to a much bigger chunk of the class documentation.

Let's do a separate pass on that?

Reply:
And while I could sed replace all of these, seems like the kind of thing doctool is supposed to do?

Reply:
I did a search for `[constant ` in all .xml files and it seems this is already used in a lot of places. But I do agree that doing a pass over all documentation pages to look for things like this would be good.

Reply:
Raised it as a draft (https://github.com/godotengine/godot/pull/60275).